cmd/global-heal.go

```diff
@@ -111,6 +111,23 @@ func healErasureSet(ctx context.Context, setIndex int, xlObj *xlObjects) error {
 	return nil
 }
 
+// deepHealObject heals given object path in deep to fix bitrot.
+func deepHealObject(ctx context.Context, objectPath string) {
+	// Get background heal sequence to send elements to heal
+	var bgSeq *healSequence
+	var ok bool
+	for {
+		bgSeq, ok = globalBackgroundHealState.getHealSequenceByToken(bgHealingUUID)
+		if ok {
+			break
+		}
+		time.Sleep(time.Second)
+	}
+
+	bgSeq.settings = madmin.HealOpts{ScanMode: madmin.HealDeepScan}
```

Comment:
Modifying the setting here means that all subsequent background healing will be deep but we do want to avoid that.

(Each 30 months, all objects & buckets are listed and sent to this background healing, but the scan mode is Normal and not deep.

I think sourceCh should receive the object path and the type of scan

Reply:
@vadmeste  I think you meant `Every 30 days`

Reply:
> @vadmeste I think you meant `Every 30 days`

oups yes right

Reply:
> I think sourceCh should receive the object path and the type of scan

Added this support